homeassistant/components/nsw_fuel_station/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "nsw_fuel_station",
   "name": "NSW Fuel Station Price",
   "documentation": "https://www.home-assistant.io/integrations/nsw_fuel_station",
-  "requirements": ["nsw-fuel-api-client==1.0.10"],
```

Comment:
Version bump fixes an inconsistency with station codes being a string instead of an int:

https://github.com/nickw444/nsw-fuel-api-client/pull/6